linux/open_vm_tools/pre_ovt_install.yml

```diff
@@ -0,0 +1,94 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+#
+- name: "Set fact to update VMware Tools by default"
+  ansible.builtin.set_fact:
+    update_vmtools: true
+  when: update_vmtools is undefined
+
+- name: "Skip test case because VM has open-vm-tools installed and update_vmtools is false"
+  include_tasks: ../../common/skip_test_case.yml
+  vars:
+    skip_msg: "Test case '{{ ansible_play_name }}' is skipped because VM has open-vm-tools and update_vmtools is set to: {{ update_vmtools }}"
+    skip_reason: "Skipped"
+  when:
+    - vmtools_is_installed
+    - not update_vmtools
+
+- name: "Test setup"
```

Comment:
This task should be put in the front of others since `vmtools_is_installed` is used.